DayCare/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.contrib.auth.models import User
+from .validators import validate_username_already_exist
+from django.core.validators import validate_email
+from django.core.validators import DecimalValidator
+from django.core.validators import MaxLengthValidator
+from django.utils import timezone
+
+
+class daycare(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, default=None, blank=True, null=False, editable=True)
+    name = models.CharField(max_length=20, blank=True, unique=True, validators=[MaxLengthValidator])
+    description = models.TextField(blank=True, null=True)
+    price_per_day = models.DecimalField(decimal_places=2, max_digits=5, validators=[DecimalValidator],
+                                        blank=False, null=False, default=0)
+    capacity = models.IntegerField(null=False, blank=True)
+    area = models.CharField(max_length=20, blank=True, validators=[MaxLengthValidator])
+    city = models.CharField(max_length=20, blank=True, validators=[MaxLengthValidator])
+    address = models.CharField(max_length=50, blank=True, validators=[MaxLengthValidator])
+
+    def __str__(self):
+        return self.name
+
+    @staticmethod
+    def create(email, username, password, name, description, price_per_day, capacity, area, city, address):
+        validate_email(email)
+        validate_username_already_exist(username)
+
+        new_daycare = daycare(user=User.objects.create_user(email=email, username=username, password=password,
+                                                            last_login=timezone.now()),
```

Comment:
Why is it necessary to set last login?
I am not sure that the face you created a user it means it just logged in, isnt it?

Reply:
deleted, the bug fixed